pkg/network/tags_linux.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/network/protocols/http"
 )
 
+const tlsTagsMask = http.GnuTLS | http.OpenSSL | http.Go
```

Comment:
consider adding a comment to `http_types.go` a comment above the definition of the tags, so people will add here their new tags, or move the definition of `tlsTagMask` to `http_types.go`